playtron-os-files/playtron-os-files.spec

```diff
@@ -1,5 +1,5 @@
 Name: playtron-os-files
-Version: 0.16.0.37
+Version: 0.16.1
```

Comment:
The version of this package is no longer tied to the OS version. Switch to standard semver.